docs/dev-ref/bug-bounty.md

```diff
@@ -1,16 +1,12 @@
 ---
-title: UMA Bug Bounty Program
-sidebar_label: Bug Bounty Program
+title: Audit & Bug Bounty Program
```

Comment:
when clicking on this page, it still highlights `Developers` header. You'll need to change the regex I think

Reply:
I went through each sidebar tab and made sure the highlighting is accurate. It looks a little messy right now in the navbar file but I will update it once we do more work to update the actual file path and redirects but I think this is a safe way to get it right in the short term.